overlay.go

```diff
@@ -723,13 +650,15 @@ func FutexFilePath(os string) (string, error) {
 //
 // For the implementation details, see https://cs.opensource.google/go/go/+/master:src/runtime/mem.go .
 func MemoryFilePath(os string) (string, error) {
-	if err := checkReplacementFileAvailability(os); err != nil {
-		return "", err
-	}
+	return replacementFilePath("MemoryFilePath", "runtime/cgo", os, "hitsumabushi_mem_linux.c")
+}
 
-	dir, err := goPkgDir("runtime/cgo", os)
-	if err != nil {
-		return "", err
-	}
-	return filepath.Join(dir, "hitsumabushi_mem_linux.c"), nil
+// CPUFilePath returns a C file's path for the CPU functions.
+// The file includes this function:
+//
+//   - int32_t hitsumabushi_getproccount()
+//
+// The default implementation uses the hardcoded cfg.NumCPU
```

Comment:
> The default implementation uses the hardcoded cfg.NumCPU

"The default implementation uses the value of a `NumCPU` option", as cfg.NumCPU is an implementation detail.

Reply:
@samhocevar Could you update this? (This is the last request...)